app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -44,7 +44,7 @@ class DatabaseMigrationAndroidTest {
   val expectedException = ExpectedException.none()
 
   @Inject
-  lateinit var migrations: ArrayList<Migration>
+  lateinit var migrations: List<@JvmSuppressWildcards Migration>
```

Comment:
`@JvmSuppressWildcards` wasn't aware of this. BTW, why are we suppressing wild cards here?

Reply:
It's related to Kotlin generics. This link explains the problem: https://stackoverflow.com/a/45384682